### PR TITLE
Update Splitbutton.html

### PR DESCRIPTION
**Pull request by helmut8080**
Adapted for Bootstrap 4: https://getbootstrap.com/docs/4.3/components/dropdowns/#split-button

### CODE REVIEW

**Comment by helmut8080** [comment]
Only the commit from May 19, 2019 should be merged. Sorry still have difficulties with Github

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/SplitButton.html [hunk 1]
```diff
@@ -4,13 +4,16 @@
...
     <body>
         <wicket:panel>
-            <button type="button" class="btn" wicket:id="button"></button>
-            <button type="button" class="btn dropdown-toggle" wicket:id="caret" data-toggle="dropdown"></button>
-            <ul wicket:id="dropdown-menu" class="dropdown-menu">
-                <li wicket:id="buttons">
-                    <a wicket:id="button"></a>
-                </li>
-            </ul>
+            <button type="button" wicket:id="button"></button>
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/SplitButton.html]: why did you remove `class="btn"` ?

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/SplitButton.html [hunk 2]
```diff
@@ -4,13 +4,16 @@
...
+            <button type="button" class="dropdown-toggle dropdown-toggle-split" wicket:id="caret" 
+                    data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                <span class="sr-only">Toggle Dropdown</span>
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/SplitButton.html]: `Toggle Dropdown` should be i18n-ed.

**Review by martin-g — CHANGES_REQUESTED** [review]

**Comment by martin-g** [comment]
> Only the commit from May 19, 2019 should be merged. Sorry still have difficulties with Githu

You need to pull the latest version of this branch and push the merge commit. This should remove the extra commits.

**Comment by martin-g** [comment]
Thank you, @helmut8080 !

**Pull request merged**